homework/CSharpOOP2/Car.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    public class Car //Task_1
+    {
+        public string Brand; //Task_2
+        public string _Color;
+        public int YearOfIssue;
+        public double _DiskSize;
+        public double _Price;
+        public static string Type { get; } ="Transport";  //Task_17
+
+        public static void GreetCar() => Console.WriteLine("Hello!"); //Task_18
+
+        public double _defaultPrice = 30.000;  //Task_13
+        public double Price
+        {
+            get
+            {
+                return _Price;
+            }
+            set
+            {
+                _Price = _defaultPrice;
+            }
+        }
+        /*public double YearOfIssue { get => _YearOfIssue = 2000; } *///Task_14
+        public double DiskSize { set => _DiskSize = 7; }   //Task_14
+
+        public string Color { get; internal set; } //Task_16
+
+
+        public void CheckBrand() //Task_3
+        {
+            if (Brand.Equals("BMW"))
+            {
+                Console.WriteLine("Car brand - BMW");
```

Comment:
I can see here a duplication :)
you can use interpolation, because you know that if brand equals BMW - you will put BMW on console :)